lib/index.js

```diff
@@ -7,8 +7,8 @@ var formatters = require('./formatters/index.js');
 
 function register (server, options, next) {
   var pool = options.pool;
-  var getAllPeople = function (cb) {
-    return query(queries.getAllPeople, pool, cb);
+  var getAllPeople = function (active, cb) {
+    return query(queries.getAllPeople(active), pool, cb);
```

Comment:
Sorry this is a bit late in the day. But what do you think about labeling these parameters as `activeOnly` or something  like that? It is more clear because if activeOnly is false, we know we will be getting active and inactive tags, whereas if `active` is false, it suggests it will only get rows where active is set to false..

It is a small change, but should be more understandable to someone new looking at the code.